serverless/service_mesh/serverless-ossm-without-kourier.adoc

```diff
@@ -0,0 +1,32 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-without-kourier"]
+= Integrating {ProductShortName} and {ServerlessProductName} without Kourier
+:context: serverless-ossm-without-kourier
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Using {ProductShortName} with {ServerlessProductName} enables developers to configure additional networking and routing options that are not supported when using {ServerlessProductName} with the default Kourier ingress implementation.
+
+[IMPORTANT]
+====
+{ServerlessProductName} only supports the use of {ProductName} functionality that is explicitly documented in this guide, and does not support other undocumented features.
+====
+
+[id="prerequisites_serverless-ossm-without-kourier"]
+== Prerequisites
+
+* The {ServerlessOperatorName} is installed on your cluster.
++
+[IMPORTANT]
+====
+Do not install the Knative Serving component before completing the following procedures. There are additional steps required when creating the `KnativeServing` custom resource defintion (CRD) to integrate Knative Serving with {ProductShortName}, which are not covered in the general Knative Serving installation procedure of the _Administration guide_.
+====
+* xref:../../service_mesh/v2x/installing-ossm.adoc#installing-ossm[{ProductName}] version 2.0.5 or newer is installed on your cluster.
+
+:FeatureName: mTLS
```

Comment:
nit: We should maybe make this `mTLS in Openshift Serverless` as this renders as 

> mTLS is a Technology Preview feature only

Which is a little weird.